Odata-docs/client/v6/batch-operations.md

```diff
@@ -164,3 +164,102 @@ The request Payload is as following:
 	--changeset_b98a784d-af07-4723-9d5c-4722801f4c4d--
 	--batch_06d8a02a-854a-4a21-8e5c-f737bbd2dea8--
 ```
+
+## Relative Uris in Batch Requests
+There are instances where an odata service is behind a load balancer which usually rewrites the outer Uri but not the internal URIs such that the request going out from the client looks like:
+
+```html
+	POST https://myservice.com/$batch HTTP/1.1
+
+	--batch_xyz
+	GET https://myservice.com/People HTTP/1.1
+	Content-Type: application/http
+	Content-Transfer-Encoding: binary
+	OData-Version: 4.0
+```
+
+BUT when it hits the load balancer it's rewritten to
+```html
+	POST https://loadbalancer/$batch HTTP/1.1 // changed to loadbalancer uri
+
+	--batch_xyz
+	GET https://myservice.com/People HTTP/1.1 // uri in the body remains unchanged
+	Content-Type: application/http
+	Content-Transfer-Encoding: binary
+	OData-Version: 4.0
+```
+
+While the odata service actually expected
+```html
+	POST https://loadbalancer/$batch HTTP/1.1
+
+	--batch_xyz
+	GET https://loadbalancer/People HTTP/1.1
+	Content-Type: application/http
+	Content-Transfer-Encoding: binary
+	OData-Version: 4.0
+```
+
+But since we cant change the internal batch uri to the load balancer uri, we instead get around the problem by using relative uri's to fix this as follows
```

Comment:
```suggestion
But since we can't change the internal batch URI to the load balancer URI, we instead get around the problem by using relative URI's to fix this as follows
```